### DISCUSSION TITLE
Why rutracker.org always get ads? Other sites it blocks ads good. (only using default filters)

### DISCUSSION THREAD

**Discussion by goodstuff626** [uBlockOrigin/uBlock-issues#3497] [Q&A]
Why does rutracker.org always get ads? Other sites ublock origin blocks ads good. (only using default filters) Is there something I'm not understanding why ublock origin always lets adds through only on rutracker?

**Comment by stephenhawk8054** [comment]
Enable "Ru Adlist" list and test again
